documentation/README.md

```diff
@@ -1523,6 +1525,26 @@ nth(1, [1, 2, 3]); // => 2
 </sup></div>
 
 
+### omit
+
+Creates a copy of the `object` without the given `props`.
+
+```js
+const omit = require('1-liners/omit');
+
+const object = {foo: 1, bar: 2, baz: 3};
+
+omit(['foo', 'baz'], object);  // => {bar: 2}
+object;                        // => {foo: 1, bar: 2, baz: 3}
```

Comment:
You don't have to state that object doesn't change here. All 1-liner functions work in an immutable way.

> Each function shall deal with data in an immutable way
– no more debugging nightmares.